merfi/backends/gpg.py

```diff
@@ -30,19 +30,20 @@ class Gpg(base.BaseBackend):
     def sign(self):
         logger.info('Starting path collection, looking for files to sign')
         repos = RepoCollector(self.path)
-        paths = repos.debian_release_files
 
-        if paths:
-            logger.info('%s matching paths found' % len(paths))
+        if repos:
+            logger.info('%s repos found' % len(repos))
             # FIXME: this should spit the actual verified command
             logger.info('will sign with the following commands:')
             logger.info('gpg --batch --yes --armor --detach-sig --output Release.gpg Release')
             logger.info('gpg --batch --yes --clearsign --output InRelease Release')
         else:
             logger.warning('No paths found that matched')
 
-        for path in paths:
-            self.sign_release(path)
+        for repo in repos:
+            # Debian "Release" files:
+            for path in getattr(repo, 'releases', set()):
```

Comment:
would you be terribly opposed at implementing those methods in the classes for convenience? As much as I dislike inheritance, maybe a parent class with all those would be nice too, that way you don't need to define them again in the two classes.

That would eliminate the `getattr()` call on these, and you can just assume these will have them with items or empty